pkg/statsd/handler_cloud.go

```diff
@@ -314,7 +418,7 @@ func updateInplace(tags *gostatsd.Tags, hostname *string, instance *gostatsd.Ins
 		// Update hostname inplace
 		*hostname = instance.ID
 		// Update tag list inplace
-		*tags = append(*tags, "region:"+instance.Region)
+		*tags = append(*tags, "region:"+instance.Region) // TODO: memory allocation cleanup
```

Comment:
This is a fixed value and should be pushed in to instance.Tags.  Didn't want to clutter this PR.  Will try and do some serious memory profiling at some point in the future because I think it's pretty rough.